test-configs.yaml

```diff
@@ -183,27 +183,47 @@ test_plans:
     params:
       job_timeout: '120'
 
-  ltp-crypto:
+  ltp-opts: &ltp-rootfs
     rootfs: debian_buster-ltp_nfs
     pattern: 'ltp/{category}-{method}-{protocol}-{rootfs}-ltp-template.jinja2'
     filters:
       - blocklist: *kselftest_defconfig_filter
-    params:
+    params: &ltp-params
       skip_install: "true"
-      tst_cmdfiles: "crypto"
+      tst_cmdfiles: ""
       job_timeout: '15'
       skipfile: skipfile-lkft.yaml
 
+  ltp-crypto:
+    <<: *ltp-rootfs
+    params:
+      <<: *ltp-params
+      tst_cmdfiles: "crypto"
+
   ltp-ipc:
-    rootfs: debian_buster-ltp_nfs
-    pattern: 'ltp/{category}-{method}-{protocol}-{rootfs}-ltp-template.jinja2'
-    filters:
-      - blocklist: *kselftest_defconfig_filter
+    <<: *ltp-rootfs
     params:
-      skip_install: "true"
+      <<: *ltp-params
       tst_cmdfiles: "ipc"
-      job_timeout: '15'
-      skipfile: skipfile-lkft.yaml
+
+  ltp-lib:
```

Comment:
Like `ltp-timers`, there are known issues because the test binaries appear to be missing from the rootfs (and the path to run the test with `runltp` may be wrong too).  So that could also be done with a separate PR after `ltp-mm`.